MapleServer2/Packets/StatPacket.cs

```diff
@@ -0,0 +1,104 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets
+{
+    public static class StatPacket
+    {
+        public static Packet SetStats(IFieldObject<Player> player)
+        {
+            return PacketWriter.Of(SendOp.STAT)
+                .WriteInt(player.ObjectId)
+                .WriteByte()
+                .WriteByte(0x23)
+                .Write(player.Value.Stats);
+        }
+
+        public static Packet UpdateMobStats(IFieldObject<Mob> mob, int damage)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STAT);
+            pWriter.WriteInt(mob.ObjectId);
+            pWriter.WriteByte();
+            pWriter.WriteByte(1);
+            pWriter.WriteByte(4); // value
+            // Stats 
+            pWriter.WriteLong(mob.Value.stats.Hp.Total);
+            pWriter.WriteLong(mob.Value.stats.Hp.Max);
+            pWriter.WriteLong(mob.Value.stats.CurrentHp.Min -= damage);
```

Comment:
Manipulation of mob stats should not happen in this method

Reply:
Should be in another class like `Damage` and call a method with the new stats value?

Reply:
Whatever calls this method should be doing the mutation

Reply:
Sure!